examples/tutorials/colabs/replay_tutorial.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
This is an auto-generated file. Don't review it. Review the py file below instead.